packages/notebook/style/index.css

```diff
@@ -32,14 +32,14 @@
   display: flex;
   flex-direction: column;
   height: 100%;
+  min-width: 240px !important;
```

Comment:
Why was the `!important` needed?

Reply:
The `jp-Document` class gets applied later by the docmanager. This has a default `min-width` and `min-height` of `120px` by `120px`. Those defaults seemed reasonable, but not for the notebook.

Reply:
Ah, how about a more specific selector here that includes `.jp-Document`?